appveyor.yml

```diff
@@ -6,11 +6,6 @@ matrix:
   fast_finish: true
 
 environment:
-  NVM_HOME: C:\Users\appveyor\AppData\Roaming\nvm
-  POWERSHELL_GALLERY_API_KEY:
-    secure: Y7B8n8Ott858qlg1xt1Kz7VnvR59aO963p5dquIEQsMUZpvK/Ip5Rmj7G/jBKM6r
-  GITHUB_ACCESS_TOKEN:
-    secure: cEpuAn4TbJ17Y4WMMsHj46T4DLFMGE4n7xafkbiVQL4jueqc9TGG7NHWg4qbwmt7

```

Comment:
Why were these removed? Are they now being set outside of source control, in the AppVeyor console?

Reply:
AppVeyor merges what's in the UI with what's in your `appveyor.yml` so these aren't actually needed here.